src/effects.cpp

```diff
@@ -1,626 +1,607 @@
+            Effect<FireFanEffect>(HeatColors_p, NUM_LEDS, 1, 12, 400, 2, NUM_LEDS / 2, Sequential, true, false),
+            Effect<FireFanEffect>(GreenHeatColors_p, NUM_LEDS, 1, 10, 400, 2, NUM_LEDS / 2, Sequential, true, false),
+            Effect<FireFanEffect>(BlueHeatColors_p, NUM_LEDS, 1, 10, 400, 2, NUM_LEDS / 2, Sequential, true, false),
+            Effect<FireFanEffect>(RainbowColors_p, NUM_LEDS, 1, 10, 400, 2, NUM_LEDS / 2, Sequential, true, false),
+            Effect<FireFanEffect>(HeatColors_p, NUM_LEDS, 1, 10, 400, 2, NUM_LEDS / 2, Sequential, true, false, true),
+            Effect<BouncingBallEffect>(3, true, true, 1),
+            Effect<RainbowFillEffect>(60, 0),
+            Effect<ColorCycleEffect>(Sequential),
+            Effect<PaletteEffect>(RainbowColors_p, 4, 0.1, 0.0, 1.0, 0.0),
+            Effect<MeteorEffect>(20, 1, 25, .15, .05),
+            Effect<MeteorEffect>(12, 1, 25, .15, .08),
+            Effect<MeteorEffect>(6, 1, 25, .15, .12),
+            Effect<MeteorEffect>(1, 1, 5, .15, .25),
+            Effect<MeteorEffect>()
+        );
+    #endif
+
+    #if defined(EFFECTS_PLATECOVER)
+        // Plate cover effect set
+        RegisterAll(*g_ptrEffectFactories,
+            Effect<ColorFillEffect>("Solid White", CRGB::White, 1),
+            Effect<ColorFillEffect>("Solid Red",   CRGB::Red,   1),
+            Effect<ColorFillEffect>("Solid Amber", CRGB(255, 50, 0), 1),
+            Effect<FireFanEffect>(HeatColors_p, NUM_LEDS, 4, 5.0, 200, 8, 8, Sequential, true, true, true, 90),
+            Effect<RainbowFillEffect>(16, 3, true),
+            Effect<MeteorEffect>(2, 1, 15, .75, .75),
+            Effect<ColorFillEffect>("Off", CRGB::Black, 1)
+        );
+    #endif
+
+    #if defined(EFFECTS_SPIRALLAMP)
+        // Spiral lamp effect set
+        RegisterAll(*g_ptrEffectFactories,
+            Effect<VUMeterVerticalEffect>(),
+            Effect<MeteorEffect>(4, 4, 10, 1.0, 1.0),
+            Effect<ColorFillEffect>("Solid White", CRGB::White, 1),
+            Effect<FireFanEffect>(HeatColors_p,      NUM_LEDS, 1, 2.5, 200, 2, 15, Sequential, true, false),
+            Effect<FireFanEffect>(GreenHeatColors_p, NUM_LEDS, 1, 2.5, 200, 2, 15, Sequential, true, false),
+            Effect<FireFanEffect>(BlueHeatColors_p,  NUM_LEDS, 1, 2.5, 200, 2, 15, Sequential, true, false),
+            Effect<FireFanEffect>(RainbowColors_p,   NUM_LEDS, 1, 2.5, 200, 2, 15, Sequential, true, false),
+            Effect<FireFanEffect>(HeatColors_p,      NUM_LEDS, 1, 2.5, 200, 2, 15, Sequential, true, false, true),
+            Effect<RainbowFillEffect>(120, 0),
+            Effect<PaletteEffect>(RainbowColors_p, 4, 0.1, 0.0, 1.0, 0.0),
+            Effect<BouncingBallEffect>(3, true, true, 8),
+            Effect<StarryNightEffect<MusicStar>>("Rainbow Twinkle Stars", RainbowColors_p, kStarryNightProbability, 1, LINEARBLEND, 0.0, 0.0, kStarryNightMusicFactor),

```

Comment:
It might be a win to "Rename Symbol" of StarryNightEffect to just StarEffect<>....

Reply:
That's entirely your call. It's been called StarryNightEffect forever, and I was thinking that maybe you were attached to the name.

But, happy to change it if that's what you want.